apps/clearance_ui/src/components/clearance_inspector/LicenseConclusions.tsx

```diff
@@ -36,7 +36,7 @@ const LicenseConclusions = ({ purl, fileSha256 }: LicenseConclusionsProps) => {
                 <div className="flex h-full w-full flex-col items-start p-1">
                     <Label className="clearance-label">Concluded license</Label>
                     {data.licenseConclusions?.length > 0 ? (
-                        <p className="h-full w-full overflow-y-auto rounded-md border p-1 text-xs">
+                        <p className="h-full w-full overflow-y-auto p-1 text-xs">
                             {data.licenseConclusions.map((license) => (
```

Comment:
Same here, should this be a list?

Reply:
Semantically, yes, but as this seems to be addressed by Johanna's PR which is awaiting for more urgent issues, I'd leave it unchanged for now.